Library/Format.swift

```diff
@@ -3,6 +3,9 @@ import Foundation
 import KsApi
 import Prelude
 
+// swiftlint:disable:next force_unwrapping
+public let UTCTimeZone = TimeZone(secondsFromGMT: 0)!
```

Comment:
i'd be down to add this and a lil helper function to `SharedFunctions.swift` in order to reuse the estimated delivery logic. it does seem that this logic is highly specific to only estimated deliveries since that is pinned to the first day of a month which is time zone specific.

whatdya think @cdolm92 and @stephencelis ?